app/views/directives/osc-persistent-volume-claim.html

```diff
@@ -186,6 +193,11 @@
                Can't be greater than {{limits.max | usageWithUnits : 'storage'}}.
              </span>
            </div>
+           <div ng-if="persistentVolumeClaimForm.capacity.$error.underStorage" class="has-error">
+             <span class="help-block">
+               Storage quota will be exceeded - Max allocated: {{storageQuota.maxStorage}} Currently used: {{storageQuota.claimStorageUsed}}.
```

Comment:
@zherman0 what is the status of the new logic? I am not seeing a commit for this